src/test/lib/ApiTest.cpp

```diff
@@ -2773,6 +2773,78 @@ void QuicTestGlobalParam()
     }
 #endif
 
+#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
```

Comment:
The stuff you added to msquic.h isn't preview (which is fine) so this doesn't need to be either.